### ISSUE TITLE
When capturing_stdout also print the stdout in case of error

### ISSUE THREAD

**Issue by joecks** [dotdoom/fastlane-plugin-flutter#31]
When  `capturing_stdout:true` is set and the flutter command fails, we also need to know what the stdout contained in order to understand the context of the issue. 

```ruby
          unless wait_thread.value.success? || (ignore_error == true)
            UI.shell_error!(<<-ERROR)
The following command has failed:
$ #{command}
[#{wait_thread.value}]

#{errors_thread.value}

Command output:
#[output]
ERROR
          end

          ignore_error
        end
```

I could also imagine to configure this with a parameter. 
This behaviour is a bit annoying when running command on the CI, and errors can not be reproduced locally.

**Comment by dotdoom** [comment]
Agree.  The general expectation is that the call site would play nicely as well and spit the output when necessary (that's what... Capturing means).  I can understand the overhead in writing that code, but I also found stdout too noisy most of the time and containing a lot of redherring distracting from actual troubleshooting.  Especially the build command.  Do you happen to have an example of what command would fail and conceal context in stdout, so that I can play with verbosity a bit to find a way to guess the intent?

Providing an argument sounds good anyway.

**Comment by joecks** [comment]
From our project: 
https://github.com/xaynetwork/xayn_discovery_app/actions/runs/2597645768
